linux_os/guide/system/bootloader-grub2/grub2_init_on_alloc_argument/rule.yml

```diff
@@ -0,0 +1,46 @@
+documentation_complete: true
+
+prodtype: rhel9
+
+title: 'Configure kernel to zero out memory before allocation (through Grub2)'
```

Comment:
The other GRUB2 rule titles don't mention `(through Grub2)`, we should keep them consistent.
Either add it to to those rules, or remove from this one. And preferably, also consistent with the zIPL rules.